corehq/apps/data_dictionary/tests/test_models.py

```diff
@@ -12,134 +10,114 @@
 )
 
 
-class TestCaseProperty(TestCase):
-    domain_name = 'test-case-property'
+def test_valid_date():
+    valid_date = '2025-11-02'
+    # check_validity raises exception on invalid date so just
+    # calling it and not getting an exception indicates validity
+    prop = CaseProperty(data_type=CaseProperty.DataType.DATE)
+    prop.check_validity(valid_date)
```

Comment:
So, one piece of feedback Biyeun left me was that standalone functions in model or view classes can tend to get lost over time -- it's not clear what functionality they belong to, so they can get shuffled around. I agreed with this feedback, but I wonder if that also should extend to tests on generic modules like models. Testing this module might involve testing for several models, and I could see it being hard to follow when they're all standalone functions like these. But then again, I'm also most comfortable with the class-based test syntax, so just throwing it out there